superdesk/publish/publish_content.py

```diff
@@ -78,9 +79,18 @@ def transmit_items(queue_items):
             logger.exception(ex)
```

Comment:
Instead of just `logger.exception(ex)` can we have meaningful log message (may be include the queue id). It would help while analyzing log files.